packages/snap-preact-components/src/components/Organisms/Recommendation/readme.md

```diff
@@ -29,6 +29,19 @@ The `loop` prop enables 'infinite' looping through the result set when swiping o
 <Recommendation controller={controller} loop={true} />
 ```
 
+### results
+The `results` prop specifies a reference to the results store array. This will override any results specified in `children` and `controller.store.results`
```

Comment:
Based on line 185 of Recommendation.tsx, it doesn't seem like this overrides results specified in `children`.

When `children` are utilized, the results data from `results` or `controller.store.results` are used for tracking purposes - which is why it is important that the lengths match that of the `children`. This is pretty important and may need to be mentioned here.